Content.Server/DeltaV/AACTablet/AACTabletSystem.cs

```diff
@@ -0,0 +1,44 @@
+using Content.Server.Chat.Systems;
+using Content.Shared.DeltaV.AACTablet;
+using Content.Shared.DeltaV.QuickPhrase;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+
+namespace Content.Server.DeltaV.AACTablet;
+
+public sealed class AACTabletSystem : EntitySystem
+{
+    [Dependency] private readonly ChatSystem _chat = default!;
+    [Dependency] private readonly ILocalizationManager _loc = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] protected readonly IGameTiming Timing = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<AACTabletComponent, AACTabletSendPhraseMessage>(OnSendPhrase);
+    }
+
+    private void OnSendPhrase(EntityUid uid, AACTabletComponent component, AACTabletSendPhraseMessage message)
+    {
+        if (component.NextPhrase > Timing.CurTime)
+            return;
+
+        // the AAC tablet uses the name of the person who pressed the tablet button
+        // for quality of life
+        var senderName = Name(message.Actor);
+        var speakerName = Loc.GetString("speech-name-relay",
+            ("speaker", Name(uid)),
+            ("originalName", senderName));
+
+        var phrase = _prototypeManager.Index<QuickPhrasePrototype>(message.PhraseID);
```

Comment:
this will throw if malf client sends an invalid id, use TryIndex and return if it fails